frontend changes

maefranco opened the pull request:
removed strange symbols in contact-us.html
corrected css classes in forms in model.py
included fonts folder

jthreatt4 commented:
@maefranco i can't commit this. There are no fixtures for the about page, workshop-index or blog-index. Also the changes to the form app break the css for the fields. The template for the forms needs the html and css for the grid framework to set widths on form fields. The classes i provided are needed for the javascript widgets on the forms to work. Also the media files folder shouldn't be included in the repo.

maefranco commented:
I'm working on the fixtures for the about page, workshop-index and blog-index. Having the CSS classes in the create radio, checkboxes, and dropdown fields caused the radio, dropdown, and check boxes to be superimposed over text fields, which is why I removed those classes from those specific fields. I'll send you screenshots of how a created form looked before and after I removed those CSS classes.
I also included the media folder in gitignore
![forms_before](https://user-images.githubusercontent.com/25971844/28085453-a1253d04-6630-11e7-9fe4-f9a889a627b7.png)
![form_after](https://user-images.githubusercontent.com/25971844/28085464-ad192300-6630-11e7-93cc-1ae4e7d81245.png)

jthreatt4 commented:
I see what your saying on the checkboxes and radio buttons, but there aren't any styles for them in the template?

maefranco commented:
There is styling for them in the template, but for some reason, including the CSS classes (.form-control, etc) in the def create radio and checkbox fields in forms/models.py created text boxes as well as radio/check buttons. I'm not sure why that is, but removing the classes from those specific fields solved the problem. Hope that answers your question.

maefranco commented:
Julian, I didn't resolve the conflict regarding import statements in home/models.py prior to creating the pull request. Do I need to create a new request?

jthreatt4 commented:
no anything you push to an open pull request updates the pull request.

jthreatt4 commented on home/models.py:
```diff
@@ -4,10 +4,16 @@
 from wagtail.wagtailimages.blocks import ImageChooserBlock
 from wagtail.wagtailcore.fields import RichTextField, StreamField
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+>>>>>>> 4ea952ce95eede0524fa8421b399824fefd630b8
```
this merge conflict wasn't resolved
